source/_integrations/izone.markdown

```diff
@@ -54,6 +54,54 @@ Unit modes off, heat, cool, dry, and fan only are supported. For units fitted wi
 
 Zones have three modes available, closed, open, and auto. These are mapped to Home Assistant modes off, fan only, and auto, respectively. Only the auto mode supports setting the temperature.
 
+## Control zone (climate control mode)
+
+With multiple climate controlled zones you can't set the target temperature of the control, but set the target temperature
+for each individual zone.
+
+The climate controller then selects the zone that is furthest away from the target and feeds the current temperature and
+target temperature into the air conditioner unit, closing any other zones that have already reached their target.
+
+In this mode the current control zone that has been selected is reported, as is the readonly target temperature for that 
+zone (read only, set the value via the individual zones). The current temperature will also be that of the control
+zone.
+
+You can add configure to read these values into sensors, along with the supply temperature (use the ID of your unit):
+
+{% raw %}
+
+```yaml
+sensor:
```

Comment:
I think it is unclear for the reader where this YAML snippet should go.

Reply:
Additional comments added to explain where it goes.